packages/stacking/src/constants.ts

```diff
@@ -0,0 +1,21 @@
+export enum StackingErrors {
```

Comment:
could we have a StackingError class with the error code, type (const), and a human-readable message? With that, developers could do `err.getMessage()` and simply display inside their app. If we just return the string, there's an opportunity for the developer to misinterpret the error string and/or misrepresent what it means to the user

Reply:
Added a function to address this.